demo_manager_node:
  ros__parameters:
    tick_hz: 2.0
    area_a_threshold: 5.0
    area_b_threshold: 10.0

robot_simulator_node:
  ros__parameters:
    initial_x: 0.0
    initial_y: 0.0
    initial_theta: 0.0
    update_rate: 50.0
